ports/voidstranger/README.md

```diff
@@ -0,0 +1,81 @@
+# README
+
+GitHub for this port- https://github.com/EzDzzIt/vsaarch64
+ 
+This port supports v1.1.1 of the Steam version of the game, or v1.1.1 of the itch.io version (itch version is patched to be equivalent to the Steam version).
+
+## Instructions for Running
+
+-Purchase game via https://store.steampowered.com/app/2121980/Void_Stranger/ 
+
+-Place all game .png, .dat, .csv, and .win files in the "/gamedata/" folder. 
+
+-On first run, the game will take a 2-4 minutes to load. The port is running through patching the data.win file via xdelta, zipping audio files and other files into the .apk, and parsing the .csv game data file. Subsequent starts should go faster. 
+
+## Controls
```

Comment:
Use markdown tables instead of just pasting the gptokeyb file. See other ports for examples.

Reply:
done